### PR TITLE
Amls 5010 - getMessage instead of toString

### PR DESCRIPTION
**Pull request by slavvco**
Implemented getMessage helper method to use it for alphabetical sorting.

## Related / Dependant PRs?

- none

## How Has This Been Tested?

- manually, unit tests 

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/asp/summary.scala.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
 @(f: forms.Form2[_],model: Asp)(implicit request: Request[_],m:Messages, lang: Lang)
 
 @header = {
-    @heading("title.cya", "summary.asp")
+@heading("title.cya", "summary.asp")
```
Comment (owen-opencast) [inline app/views/asp/summary.scala.html]: Indent?

File: app/views/tradingpremises/summary_details.scala.html [hunk 2]
```diff
@@ -21,111 +21,112 @@
 @import models.businessmatching.MoneyServiceBusiness
 @import utils._
 
-@(model: TradingPremises, isMsb: Boolean, index: Int, hasOneService: Boolean, hasOneMsbService: Boolean = None)(implicit request: Request[_],m:Messages, lang: Lang)
+@(model: TradingPremises, isMsb: Boolean, index: Int, hasOneService: Boolean, hasOneMsbService: Boolean = None)(implicit request: Request[_], m: Messages, lang: Lang)
 
 @header = {
-    @heading("summary.checkyouranswers.title", "summary.tradingpremises")
+@heading("summary.checkyouranswers.title", "summary.tradingpremises")
```
Comment (owen-opencast) [inline app/views/tradingpremises/summary_details.scala.html]: Indent?

File: app/views/tradingpremises/summary_details.scala.html [hunk 3]
```diff
@@ -21,111 +21,112 @@
...
+        }
 
         @checkYourAnswers(
             question = Messages("tradingpremises.summary.address"),
             editLinkTag = "tradingpremisedsummaryaddress-edit",
             editUrl = controllers.tradingpremises.routes.WhereAreTradingPremisesController.get(index, true).toString
         ) {
             <p>@yta.tradingPremisesAddress.toLines.map { l =>
-                @l<br />
+                @l <br />
```
Comment (owen-opencast) [inline app/views/tradingpremises/summary_details.scala.html]: Is this intentional? Does it render out a space at the end of the line?

File: app/models/tradingpremises/TradingPremisesMsbServices.scala [hunk 4]
```diff
@@ -17,21 +17,37 @@
...
+    }
+}
 
 case object TransmittingMoney extends TradingPremisesMsbService
 case object CurrencyExchange extends TradingPremisesMsbService
 case object ChequeCashingNotScrapMetal extends TradingPremisesMsbService
 case object ChequeCashingScrapMetal extends TradingPremisesMsbService
 case object ForeignExchange extends TradingPremisesMsbService
 
+
```
[the remark this answers is not in the document]
Comment (slavvco) [reply app/models/tradingpremises/TradingPremisesMsbServices.scala]: done

File: app/views/businessmatching/summary.scala.html [hunk 5]
```diff
@@ -47,7 +47,7 @@
             question = Messages("businessmatching.summary.business.address.lbl")
         ) {
             <p>@review.businessAddress.toLines.map { line =>
-                @line<br />
+                @line <br />
```
[the remark this answers is not in the document]
Comment (slavvco) [reply app/views/businessmatching/summary.scala.html]: removed, looks like not needed

File: app/views/asp/summary.scala.html (hunk 1 above)
Comment (slavvco) [reply app/views/asp/summary.scala.html]: done

File: app/views/tradingpremises/summary_details.scala.html (hunk 2 above)
Comment (slavvco) [reply app/views/tradingpremises/summary_details.scala.html]: done

File: app/views/tradingpremises/summary_details.scala.html (hunk 3 above)
Comment (slavvco) [reply app/views/tradingpremises/summary_details.scala.html]: looks like not needed

**Pull request merged**